inc/lift/Request.h

```diff
@@ -241,7 +241,18 @@ class Request {
      * @param verify the verify value to set the CURLOPT_SSL_VERIFYHOST option to
      */
     auto SetVerifySSLHost(int64_t verify) -> void;
-
+    
+    /**
+     * Method to allow users to set any CURLoption with any value type
```

Comment:
I'd like a comment indicating that this should only be used if the feature isn't already a part of liblift, and ideally one should consider adding it to liblift before using this